12_upgrade/20_upgrade_centos_rhel.txt

```diff
@@ -36,6 +36,14 @@ yum update "rudder-*" "ncf*"
 
 ----
 
+and after the upgrade of these packages, restart jetty to apply the changes:
+
+----
+
+/etc/init.d/rudder-jetty restart
```

Comment:
I think it's better to use service rudder-jetty restart, si that this is correct in upstream branches. I think it works on older OS anyway.